MAUI/Maps/bubble.md

```diff
@@ -0,0 +1,407 @@
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <map:SfMaps>
+        <map:SfMaps.Layer>
+            <map:MapShapeLayer 
+                  ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json"  ShowBubbleTooltip="True"
+                  DataSource="{Binding Data}" PrimaryValuePath="Continent" ShapeDataField="continent" ShowBubbles="True" >
+
+                <map:MapShapeLayer.BubbleSettings>
+                  <map:MapBubbleSettings ColorValuePath="Population" 
+                      SizeValuePath="Population" 
+                      Fill="LightGreen"
+                      Stroke="Green"
+                      StrokeThickness="2"
+                      MinSize="30"
+                      MaxSize="80">
+                  </map:MapBubbleSettings>
+                </map:MapShapeLayer.BubbleSettings>
+            </map:MapShapeLayer>
+        </map:SfMaps.Layer>
+    </map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+Data = new ObservableCollection<Model>();
+        Data.Add(new Model("Asia", 51));
+        Data.Add(new Model("Africa", 58));
+        Data.Add(new Model("Europe", 48));
+        Data.Add(new Model("North America", 41));
+        Data.Add(new Model("South America", 14));
+        Data.Add(new Model("Australia", 23));
+        
+        SfMaps maps = new SfMaps();
+        MapShapeLayer layer = new MapShapeLayer();
+        layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+        layer.DataSource = Data;
+        layer.PrimaryValuePath = "Continent";
+        layer.ShapeDataField = "continent";
+        layer.ShowBubbleTooltip = true;
+        layer.ShowBubbles = true;
+
+        MapBubbleSettings bubbleSetting = new MapBubbleSettings()
+        {
+            ColorValuePath = "Population",
+            SizeValuePath = "Population",
+            Fill = Colors.LightGreen,
+            Stroke = Colors.Green,
+            StrokeThickness = 2,
+            MinSize = 30,
+            MaxSize =80
+            
+        };
+
+        layer.BubbleSettings = bubbleSetting;
+        maps.Layer = layer;
+        this.Content = maps;
+    }
+   public class Model
+   {
+        public Model(string continent, int population)
+        {
+            Continent = continent;
+            Population = population;
+        }
+        public String Continent { get; set; }
+        public int Population { get; set; }
+  }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Bubble tooltip](images/bubble/bubble-tooltip.png)
+
+## Color
+
+You can customize the bubble color based on the value returned from the [`MapBubbleSettings.ColorMappings`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeSource/bubbleColorValueMapper.html) property. 
+
+If [`ColorMappings`](https://pub.dev/documentation/syncfusion_flutter_maps/latest/maps/MapShapeSource/bubbleColorValueMapper.html) returns a color, then the color will be applied to the bubble straightaway.
```

Comment:
If `ColorValuePath` provides a color, then the color will be applied to the bubble straightaway.